evm/sign-message.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const { ethers } = require('hardhat');
+require('dotenv').config();
```

Comment:
Import `cli-utils` instead which does this already. you can use it to add the private key option as well